src/help-widget.ts

```diff
@@ -205,4 +212,30 @@ export class BBHelpHelpWidget {
   private isCollapsed() {
     return this.container.classList.contains(HELP_CLOSED_CLASS);
   }
+
+  private watchWindowWidth() {
+    window.addEventListener('resize', () => {
+      this.resizeContainer();
+    });
+  }
+
+  private resizeContainer() {
+    // width pulled from $screenXsMax scss variable
+    if (window.innerWidth < 479) {
+      if (!this.container.classList.contains('bb-help-closed')) {
+        this.setElementDimensions(this.container, '100%', '100%');
+      } else {
+        this.setElementDimensions(this.container, this.containerInitialWidth, '100%');
+      }
+      this.invoker.style.display = 'none';
+    } else {
+      this.setElementDimensions(this.container, this.containerInitialWidth, this.containerInitialHeight);
+      this.invoker.style.display = 'flex';
+    }
+  }
+
+  private setElementDimensions(container: HTMLElement, width: string, height: string) {
```

Comment:
Trying to read through the logic above, but isn't this something that can be accomplished by toggling classes?

```
.bb-help-mobile-sizes: { 
   width: 100%;
  height: 100%;
 }
```

```
.bb-help-mobile-closed: {
   width: XXXpx;
   right: -XXXpx;
}
```

etc?  Or do we need to specifically change the widths here?